doc/nrf/libraries/others/memfault_ncs.rst

```diff
@@ -41,6 +41,9 @@ In addition, you must configure a Memfault project key using :kconfig:option:`CO
 
 To get access to all the benefits, like up to 100 free devices connected, register at the `Memfault registration page`_.
 
+.. note::
+   When building applications with  `Trusted Firmware M`_ (TF-M), the faults resulting from memory access in secure regions will not be caught by memfault's fault handler.
```

Comment:
I think we either should say how to use SPM instead of TF-M or link to a doc page that explains this.